2025-05-30 18:03:25.371 [info] Extension host with pid 2796 started
2025-05-30 18:03:25.449 [info] ExtensionService#_doActivateExtension vscode.configuration-editing, startup: false, activationEvent: 'onLanguage:jsonc'
2025-05-30 18:03:25.465 [info] ExtensionService#_doActivateExtension vscode.json-language-features, startup: false, activationEvent: 'onLanguage:jsonc'
2025-05-30 18:03:25.512 [info] ExtensionService#_doActivateExtension vscode.typescript-language-features, startup: false, activationEvent: 'onLanguage:jsonc'
2025-05-30 18:03:25.546 [info] ExtensionService#_doActivateExtension vscode.emmet, startup: false, activationEvent: 'onLanguage'
2025-05-30 18:03:25.743 [info] ExtensionService#_doActivateExtension vscode.git-base, startup: true, activationEvent: '*', root cause: vscode.git
2025-05-30 18:03:25.747 [info] ExtensionService#_doActivateExtension mhutchie.git-graph, startup: true, activationEvent: '*'
2025-05-30 18:03:25.781 [info] ExtensionService#_doActivateExtension vscode.git, startup: true, activationEvent: '*'
2025-05-30 18:03:25.810 [info] ExtensionService#_doActivateExtension vscode.github, startup: true, activationEvent: '*'
2025-05-30 18:03:25.846 [info] Eager extensions activated
2025-05-30 18:03:25.866 [info] ExtensionService#_doActivateExtension vscode.debug-auto-launch, startup: false, activationEvent: 'onStartupFinished'
2025-05-30 18:03:25.868 [info] ExtensionService#_doActivateExtension vscode.merge-conflict, startup: false, activationEvent: 'onStartupFinished'
2025-05-30 18:03:25.881 [info] ExtensionService#_doActivateExtension Catppuccin.catppuccin-vsc, startup: false, activationEvent: 'onStartupFinished'
2025-05-30 18:03:25.884 [info] ExtensionService#_doActivateExtension Catppuccin.catppuccin-vsc-icons, startup: false, activationEvent: 'onStartupFinished'
2025-05-30 18:03:25.941 [info] ExtensionService#_doActivateExtension vadimcn.vscode-lldb, startup: false, activationEvent: 'onStartupFinished'
2025-05-30 18:03:29.881 [info] Extension host terminating: renderer closed the MessagePort
2025-05-30 18:03:29.892 [info] Extension host with pid 2796 exiting with code 0
2025-05-30 18:03:30.748 [info] Extension host with pid 2888 started
2025-05-30 18:03:30.748 [info] Skipping acquiring lock for /home/dimgerasimou/.config/VSCodium/User/workspaceStorage/7719972917581ceffa55793e28d7dfb5.
2025-05-30 18:03:30.778 [info] ExtensionService#_doActivateExtension vscode.configuration-editing, startup: false, activationEvent: 'onLanguage:jsonc'
2025-05-30 18:03:30.785 [info] ExtensionService#_doActivateExtension vscode.json-language-features, startup: false, activationEvent: 'onLanguage:jsonc'
2025-05-30 18:03:30.828 [info] ExtensionService#_doActivateExtension vscode.typescript-language-features, startup: false, activationEvent: 'onLanguage:jsonc'
2025-05-30 18:03:30.870 [info] ExtensionService#_doActivateExtension vscode.emmet, startup: false, activationEvent: 'onLanguage'
2025-05-30 18:03:31.035 [info] ExtensionService#_doActivateExtension vscode.git-base, startup: true, activationEvent: '*', root cause: vscode.git
2025-05-30 18:03:31.037 [info] ExtensionService#_doActivateExtension mhutchie.git-graph, startup: true, activationEvent: '*'
2025-05-30 18:03:31.044 [info] ExtensionService#_doActivateExtension ms-python.python, startup: true, activationEvent: 'workspaceContains:.venv'
2025-05-30 18:03:31.190 [info] ExtensionService#_doActivateExtension vscode.git, startup: true, activationEvent: '*'
2025-05-30 18:03:31.219 [info] ExtensionService#_doActivateExtension vscode.github, startup: true, activationEvent: '*'
2025-05-30 18:03:31.297 [info] Eager extensions activated
2025-05-30 18:03:31.310 [info] ExtensionService#_doActivateExtension vscode.debug-auto-launch, startup: false, activationEvent: 'onStartupFinished'
2025-05-30 18:03:31.312 [info] ExtensionService#_doActivateExtension vscode.merge-conflict, startup: false, activationEvent: 'onStartupFinished'
2025-05-30 18:03:31.325 [info] ExtensionService#_doActivateExtension Catppuccin.catppuccin-vsc, startup: false, activationEvent: 'onStartupFinished'
2025-05-30 18:03:31.328 [info] ExtensionService#_doActivateExtension Catppuccin.catppuccin-vsc-icons, startup: false, activationEvent: 'onStartupFinished'
2025-05-30 18:03:31.391 [info] ExtensionService#_doActivateExtension vadimcn.vscode-lldb, startup: false, activationEvent: 'onStartupFinished'
2025-05-30 18:03:51.240 [info] ExtensionService#_doActivateExtension vscode.ipynb, startup: false, activationEvent: 'onNotebookSerializer:jupyter-notebook'
2025-05-30 18:03:51.347 [info] ExtensionService#_doActivateExtension luma.jupyter, startup: false, activationEvent: 'onNotebook:jupyter-notebook'
2025-05-30 18:03:52.198 [info] ExtensionService#_doActivateExtension ms-toolsai.jupyter, startup: false, activationEvent: 'onNotebook:jupyter-notebook'
2025-05-30 18:03:52.201 [info] ExtensionService#_doActivateExtension ms-toolsai.vscode-jupyter-cell-tags, startup: false, activationEvent: 'onNotebook:jupyter-notebook'
2025-05-30 18:03:52.203 [info] ExtensionService#_doActivateExtension ms-toolsai.vscode-jupyter-slideshow, startup: false, activationEvent: 'onNotebook:jupyter-notebook'
2025-05-30 18:03:52.369 [info] ExtensionService#_doActivateExtension ms-toolsai.jupyter-renderers, startup: false, activationEvent: 'api', root cause: ms-toolsai.jupyter
2025-05-30 18:03:52.903 [info] ExtensionService#_doActivateExtension ms-python.debugpy, startup: false, activationEvent: 'onLanguage:python'
2025-05-30 18:20:48.428 [error] [ms-python.python] provider FAILED
2025-05-30 18:20:48.428 [error] Error: UnboundLocalError: cannot access local variable 'start_index' where it is not associated with a value
    at ae (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1965471)
    at oe (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1963431)
    at Immediate.<anonymous> (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1959725)
    at processImmediate (node:internal/timers:483:21)
2025-05-30 18:20:54.373 [error] [ms-python.python] provider FAILED
2025-05-30 18:20:54.373 [error] Error: UnboundLocalError: cannot access local variable 'start_index' where it is not associated with a value
    at ae (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1965471)
    at oe (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1963431)
    at Immediate.<anonymous> (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1959725)
    at processImmediate (node:internal/timers:483:21)
2025-05-30 18:20:58.700 [error] [ms-python.python] provider FAILED
2025-05-30 18:20:58.700 [error] Error: UnboundLocalError: cannot access local variable 'start_index' where it is not associated with a value
    at ae (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1965471)
    at oe (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1963431)
    at Immediate.<anonymous> (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1959725)
    at processImmediate (node:internal/timers:483:21)
2025-05-30 18:21:28.535 [info] ExtensionService#_doActivateExtension vscode.extension-editing, startup: false, activationEvent: 'onLanguage:json'
2025-05-30 18:21:28.545 [info] ExtensionService#_doActivateExtension vscode.npm, startup: false, activationEvent: 'onLanguage:json'
2025-05-31 16:53:05.907 [info] ExtensionService#_doActivateExtension vscode.markdown-language-features, startup: false, activationEvent: 'onLanguage:markdown'
2025-05-31 16:53:06.230 [info] ExtensionService#_doActivateExtension vscode.markdown-math, startup: false, activationEvent: 'api', root cause: vscode.markdown-language-features
2025-05-31 18:10:28.349 [error] [ms-python.python] provider FAILED
2025-05-31 18:10:28.350 [error] Error: UnboundLocalError: cannot access local variable 'start_index' where it is not associated with a value
    at ae (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1965471)
    at oe (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1963431)
    at Immediate.<anonymous> (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1959725)
    at processImmediate (node:internal/timers:483:21)
2025-05-31 18:10:31.046 [error] [ms-python.python] provider FAILED
2025-05-31 18:10:31.046 [error] Error: UnboundLocalError: cannot access local variable 'start_index' where it is not associated with a value
    at ae (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1965471)
    at oe (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1963431)
    at Immediate.<anonymous> (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1959725)
    at processImmediate (node:internal/timers:483:21)
2025-05-31 18:10:40.978 [error] [ms-python.python] provider FAILED
2025-05-31 18:10:40.978 [error] Error: UnboundLocalError: cannot access local variable 'start_index' where it is not associated with a value
    at ae (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1965471)
    at oe (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1963431)
    at Immediate.<anonymous> (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1959725)
    at processImmediate (node:internal/timers:483:21)
2025-05-31 18:10:45.190 [error] [ms-python.python] provider FAILED
2025-05-31 18:10:45.190 [error] Error: UnboundLocalError: cannot access local variable 'start_index' where it is not associated with a value
    at ae (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1965471)
    at oe (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1963431)
    at Immediate.<anonymous> (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1959725)
    at processImmediate (node:internal/timers:483:21)
2025-05-31 20:11:37.513 [error] [ms-python.python] provider FAILED
2025-05-31 20:11:37.513 [error] Error: KeyError: <ForStmt: for i in range(1, n): 		product *= (x_pred - x_data[i - 1]) 		result += coeffs[i] * product@38,1>
    at ae (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1965471)
    at oe (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1963431)
    at Immediate.<anonymous> (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1959725)
    at processImmediate (node:internal/timers:483:21)
2025-05-31 20:11:41.301 [error] [ms-python.python] provider FAILED
2025-05-31 20:11:41.302 [error] Error: KeyError: <ForStmt: for i in range(1, n): 		product *= (x_pred - x_data[i - 1]) 		result += coeffs[i] * product@38,1>
    at ae (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1965471)
    at oe (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1963431)
    at Immediate.<anonymous> (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1959725)
    at processImmediate (node:internal/timers:483:21)
2025-05-31 20:11:53.411 [error] [ms-python.python] provider FAILED
2025-05-31 20:11:53.411 [error] Error: KeyError: <ForStmt: for i in range(1, n): 		product *= (x_pred - x_data[i - 1]) 		result += coeffs[i] * product@38,1>
    at ae (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1965471)
    at oe (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1963431)
    at Immediate.<anonymous> (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1959725)
    at processImmediate (node:internal/timers:483:21)
2025-05-31 20:12:17.811 [error] [ms-python.python] provider FAILED
2025-05-31 20:12:17.811 [error] Error: KeyError: <ForStmt: for i in range(1, n): 		product *= (x_pred - x_data[i - 1]) 		result += coeffs[i] * product@38,1>
    at ae (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1965471)
    at oe (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1963431)
    at Immediate.<anonymous> (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1959725)
    at processImmediate (node:internal/timers:483:21)
2025-05-31 20:32:02.140 [error] [ms-python.python] provider FAILED
2025-05-31 20:32:02.141 [error] Error: KeyError: <ForStmt: for i in range(1, n): 		product *= (x_pred - x_data[i - 1]) 		result += coeffs[i] * product@38,1>
    at ae (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1965471)
    at oe (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1963431)
    at Immediate.<anonymous> (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1959725)
    at processImmediate (node:internal/timers:483:21)
2025-05-31 20:34:10.505 [error] [ms-python.python] provider FAILED
2025-05-31 20:34:10.505 [error] Error: KeyError: <ForStmt: for i in range(1, n): 		product *= (x_pred - x_data[i - 1]) 		result += coeffs[i] * product@38,1>
    at ae (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1965471)
    at oe (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1963431)
    at Immediate.<anonymous> (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1959725)
    at processImmediate (node:internal/timers:483:21)
2025-05-31 21:37:52.834 [error] [ms-python.python] provider FAILED
2025-05-31 21:37:52.834 [error] Error: UnboundLocalError: cannot access local variable 'start_index' where it is not associated with a value
    at ae (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1965471)
    at oe (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1963431)
    at Immediate.<anonymous> (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1959725)
    at processImmediate (node:internal/timers:483:21)
2025-05-31 21:37:57.076 [error] [ms-python.python] provider FAILED
2025-05-31 21:37:57.077 [error] Error: UnboundLocalError: cannot access local variable 'start_index' where it is not associated with a value
    at ae (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1965471)
    at oe (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1963431)
    at Immediate.<anonymous> (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1959725)
    at processImmediate (node:internal/timers:483:21)
2025-05-31 21:47:08.961 [info] Extension host terminating: renderer closed the MessagePort
2025-05-31 21:47:09.040 [info] Extension host with pid 2888 exiting with code 0
